More documentation fixes (#75)
epinio/docs: src/howtos/install_epinio_on_minikube.md

# Creating a Minikube Kubernetes Cluster

> This page was written using the following versions:

Reviewer: ```suggestion
This how-to was written using the following versions:
```

### Install Epinio on the Cluster

> With Minikube, you will need to deploy Traefik and cert-manager.

Reviewer: ```suggestion
With Minikube, you'll need to deploy Traefik and cert-manager.
```
